components/CreateTaskForm/CreateTaskForm.js

```diff
@@ -0,0 +1,106 @@
+import styled from "styled-components";
+import { useState } from "react";
+import { useEffect } from "react";
+
+const StyledCreateButton = styled.button`
+  border: 1px solid black;
+  border-radius: 50%;
+  background: grey;
+  text-decoration: none;
+  color: white;
+  font-size: 20px;
```

Comment:
To improve scalability, I recommend using rem units instead of px for font sizes.